server/fishtest/rundb.py

```diff
@@ -353,6 +353,13 @@ def start_timer(self):
         self.timer.start()
 
     def buffer(self, run, flush):
+        if not self.is_primary_instance():
+            print(
+                "Warning: attempt to use the run_cache on the "
```

Comment:
I didn't know the automatic concatenation of string literals, it's surprising that this works:
```python
a ="0" "1"   "2"
```
This syntax is easier to my eyes but I'm open to other views:
```diff
            print(
-                "Warning: attempt to use the run_cache on the "
            print(
+                "Warning: attempt to use the run_cache on the",
```

Reply:
No strong feelings about this so I changed it. Note however that the comma trick will only work in print statements...